projects/hmpps-auth-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/UserService.kt

```diff
@@ -60,4 +67,14 @@ class UserService(
         enabled = enabled,
         roles = getUserRoles(dn)
     )
+
+    private fun LdapUser.toUserDetails(userId: Long) = UserDetails(
```

Comment:
Is it worth combining this with the method above to reduce repetition?  Or even calling this one from the other e.g. something like:
```kotlin
    private fun LdapUser.toUserDetails() = auditUserService.findUser(username)?.let { toUserDetails(it.id) } 
          ?: throw NotFoundException("User entity", "username", username)
```

Reply:
Yeah - absolutely. Should have done this. Thanks. Done